### ISSUE TITLE
NC install and documentation give conflicting webdav information

### ISSUE THREAD

**Issue by deadwax** [nextcloud/documentation#1771]
As per #698, which was closed but not fixed.

The Webdav link given in the files page of nextcloud is:
http://example.com/nextcloud/remote.php/webdav/
This works - although it gives http when it should give https - connecting to this URL works.

The current manual at:
https://docs.nextcloud.com/server/16/user_manual/files/access_webdav.html
gives instead:
example.com/nextcloud/remote.php/dav/files/USERNAME/
This does not work - zero files are found, although it connects without error.
(I'm well aware USERNAME needs to be changed to my user)

This is on NC 16.0.7

**Comment by skjnldsv** [comment]
> The Webdav link given in the files page of nextcloud is:
> [http://example.com/nextcloud/remote.php/webdav](http://example.com/nextcloud/remote.php/webdav/)
> This works - although it gives http when it should give https - connecting to this URL works.

This is the old way. All new clients are using the remote.php/dav/files/xxx and it works fine here (just tested again)
Please make sure your webserver config is up to date to our standards (see docs)

What are you using to list the files of your user?

**Comment by deadwax** [comment]
> What are you using to list the files of your user?

Two different Android sync clients - Folder Sync and Synchronize Ultimate - both only work on the first way.  I've just tried folder sync with another server not admined by me, with exactly the same results.

As far as I'm aware my server config is correct.

**Comment by skjnldsv** [comment]
Works here :)

```php
Listing collection '/remote.php/dav/files/admin/': succeeded.
Coll:   At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet          0  nov.  1 13:10
Coll:   Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua          0  déc.  9 12:42
Coll:   Mov                                    0  oct. 11 08:18
Coll:   New folder                             0  nov. 27 18:11
Coll:   Nextcloud                              0  janv.  9 09:07
Coll:   Photos                                 0  nov. 14 10:15
Coll:   System information                     0  nov. 21 08:36
Coll:   Test ?                                 0  janv.  7 17:07
Coll:   Vacations                              0  nov. 14 19:03
        Image with a # in it.png          290105  oct.  3 08:21
        U3uNihd.jpg                      1876453  oct. 31 08:02
        UqOShWC.gif                      1821397  mai  4  2018
        app.svg                              271  oct. 25 10:53
        bbb_sunflower_1080p_60fps_normal.mp4  355856562  juil. 22  2018
        bbb_sunflower_1080p_60fps_normal.png      16413  oct.  3 08:21
        bbb_sunflower_2160p_60fps_normal.mp4  673223862  nov. 14 08:20
        big_buck_bunny_480p_surround-fix-vp9.mkv  112473953  avril 29  2019
        big_buck_bunny_480p_surround-fix.mkv   62103994  avril 29  2019
        welcome.txt                            7  oct.  3 15:33
        yYjCHQG.mp4                      9749895  oct. 10 09:15

```

You probably have a configuration issue somewhere :confused:

**Comment by deadwax** [comment]
Ok, given the address specified in the files interface works I'm not terribly inclined to chase it.

We do still have a conflict between the address in documentation and the address an NC install gives users.

**Comment by skjnldsv** [comment]
Fix in https://github.com/nextcloud/server/pull/18804

**Comment by ghuls** [comment]
I noticed a similar problem, where `webdav` would work, but `/dav/files/USERNAME` wouldn't.

I figured out that `USERNAME` is not always the same than the login username as it gets resolved from a userid (used to login) to an actual username (SAML2 authentication).
  - To login to NextCloud: userid
  - Directory in `/dav/files/`:  actual username

```
userid

https://example.com/nextcloud/remote.php/dav/files/my_actual_username@example.com/
```

https://example.com/nextcloud/index.php/apps/files ==> `Settings` ==> `WebDAV`: shows the correct link (with actual username).
